src/ceph.in

```diff
@@ -287,7 +287,7 @@ daemonperf {type.id | path} list|ls [stat-pats] [priority]
     """, file=sys.stdout)
 
 
-def do_extended_help(parser, args):
+def do_extended_help(parser, args, target=None, filter=True):
```

Comment:
instead of using the default parameters, i'd suggest expose the `partial` parameter to the caller. in other words, to define

```py
def do_extended_help(parser, args, target, partial):
  # ...
  return help_for_target(target, partial)
# ... 

return do_extended_help(parser, childargs, ('mon', ''), ' '.join(childargs))
#...
return do_extended_help(parser, childargs, childargs[1].split('.', 1), None)
```